escodegen.js

```diff
@@ -2312,6 +2312,11 @@
                 return 'null';
             }
 
+            // BigInt, eg: 1n
+            if (typeof expr.bigint === 'string') {
+                return expr.raw;
```

Comment:
The `raw` field is not guaranteed to be there.

```suggestion
                return expr.bigint + 'n';
```